doc/API.md

```diff
@@ -32,26 +32,31 @@ Display general information about the current daemon state.
-| 6     | `emergency_vaulted`  | The vault has its emergency tx confirmed, funds are in the Deep Emergency Vault                              |
-| 5     | `spendable`          | The vault has its unvault tx timelock expired and can be spent                                               |
-| 6     | `spending`           | The vault has a spending tx broadcasted                                                                      |
-| 7     | `spent`              | The vault has a spending tx confirmed, the vault is spent                                                    |
+| 2     | `secured`            | The vault's emergency transaction is fully signed and shared with the watchtowers                            |
+| 3     | `active`             | The vault's unvault, cancel, and unvault-emergency txs are fully signed and shared with the watchtowers      |
+| 4     | `unvaulting`         | The vault has its unvault tx broadcasted                                                                     |
+| 5     | `unvaulted`          | The vault has its unvault tx confirmed                                                                       |
+| 6     | `cancelling`         | The vault has its cancel tx broadcasted, funds are sent to an other vault                                    |
+| 7     | `cancelled`          | The vault has its cancel tx confirmed, funds are in an other vault                                           |
+| 3 / 6 | `emergency_vaulting` | The vault has its emergency tx broadcasted, funds are sent to the Deep Emergency Vault                       |
+| 4 / 7 | `emergency_vaulted`  | The vault has its emergency tx confirmed, funds are in the Deep Emergency Vault                              |
+| 6     | `spendable`          | The vault has its unvault tx timelock expired and can be spent                                               |
+| 7     | `spending`           | The vault has a spending tx broadcasted                                                                      |
+| 8     | `spent`              | The vault has a spending tx confirmed, the vault is spent                                                    |
 
 ### Vault resource
 
-| Field    | Type   | Description                                             |
-| -------- | ------ | ------------------------------------------------------- |
-| `amount` | int    | Amount of the vault in satoshis                         |
-| `txid`   | string | Unique ID of the vault deposit transaction              |
-| `status` | string | Status of the vault ([vault statuses](#vault-statuses)) |
+| Field         | Type   | Description                                                                          |
+| ------------- | ------ | ------------------------------------------------------------------------------------ |
+| `blockheight` | int    | Block height at which the vault deposit transaction was confirmed (0 if unconfirmed) |
```

Comment:
can you keep the table sorted alphabetically please ?

Reply:
Hmm so *this* was the rationale for this order!

Reply:
Done